### DISCUSSION TITLE
[Question]弹幕服务器连接类型的区别

### DISCUSSION THREAD

**Discussion by SekiBetu** [HMBSbige/BilibiliLiveRecordDownLoader#14] [Q&A] [answered]
RT，TCP和ws之间的区别已经懂了，SecureWebsocket和ws的区别呢
这些类型中，对于直播而言哪个比较稳定呢

**Comment by HMBSbige** [comment, marked as answer]
跟 https 和 http 区别一样

理论上握手时间 wss>ws>tcp，连接以后基本一样

正常看直播都用 wss，稳不稳还是要看 B 站的服务器

wss/ws 可走 http 代理
